controller/api/public/edges.go

```diff
@@ -0,0 +1,152 @@
+package public
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	pb "github.com/linkerd/linkerd2/controller/gen/public"
+	"github.com/prometheus/common/model"
+)
+
+type edgeResult struct {
+	res *pb.Edge
+	err error
+}
+
+const (
+	inboundIdentityQuery  = "label_replace(sum(increase(response_total%s[1m])) by (%s,client_id), \"destination_%s\", \"$1\", \"%s\", \"(.+)\")"
```

Comment:
Thanks for the tip and the info on simplifying! @rmars made a good point further down that the `label_replace` isn't actually necessary since I can just query the `pod` and `destination_pod` labels directly in `processEdgeMetrics()`, which makes these queries even shorter.